study-builder/studies/osteo/study.conf

```diff
@@ -1815,6 +1830,20 @@
       "dispatchToHousekeeping": true,
       "order": 1
     },
+    {
+      "trigger": {
+        "type": "INVITATION_CREATED"
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
```

Comment:
I made some slightly different design decisions. Based on that, this should be `INVITATION_EMAIL` type instead. See PR #30.